config/initializers/devise.rb

```diff
@@ -0,0 +1,15 @@
+Devise.setup do |config|
+  config.mailer_sender = "dev@example.com"
+
+  require 'devise/orm/active_record'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.